.ci/test_r_package.sh

```diff
@@ -118,6 +118,12 @@ if [[ $OS_NAME == "macos" ]]; then
     fi
 fi
 
+# fix for issue where CRAN was not returning {lattice} when using R 3.6
+# "Warning: dependency ‘lattice’ is not available"
+if [[ "${R_MAJOR_VERSION}" == "3" ]]; then
+    Rscript --vanilla -e "install.packages('lattice', repos = 'https://mran.microsoft.com', lib = '${R_LIB_PATH}')"
+fi
```

Comment:
I don't totally understand, but it seems like CRAN (both from RStudio and the official CRAN) have stopped serving `{lattice}` to R 3.6 🤷🏻 

**official CRAN**

```shell
docker run \
    --rm \
    rocker/verse:3.6 \
    Rscript -e "install.packages('lattice', repos = 'https://cran.r-project.org')"
```

```text
Installing package into ‘/usr/local/lib/R/site-library’
(as ‘lib’ is unspecified)
Warning message:
package ‘lattice’ is not available (for R version 3.6.0)
```

**RStudio**

```shell
docker run \
    --rm \
    rocker/verse:3.6 \
    Rscript -e "install.packages('lattice', repos = 'https://cran.rstudio.com')"
```

```text
Installing package into ‘/usr/local/lib/R/site-library’
(as ‘lib’ is unspecified)
Warning message:
package ‘lattice’ is not available (for R version 3.6.0)
```

**MRAN**

```shell
docker run \
    --rm \
    rocker/verse:3.6 \
    Rscript -e "install.packages('lattice', repos = 'https://mran.microsoft.com')"
```

```text
Installing package into ‘/usr/local/lib/R/site-library’
(as ‘lib’ is unspecified)
trying URL 'https://mran.microsoft.com/src/contrib/lattice_0.20-45.tar.gz'
Content type 'application/octet-stream' length 399470 bytes (390 KB)
==================================================
downloaded 390 KB

* installing *source* package ‘lattice’ ...
...
* DONE (lattice)
```